spec/rails_helper.rb

```diff
@@ -47,8 +47,4 @@
   config.example_status_persistence_file_path = "#{::Rails.root}/tmp/persistent_examples.txt"
 
   config.filter_rails_from_backtrace!
-
-  # Tmp until we handle the multi-tenancy case where we do not know what
-  # casa org we are when we hit the sign-in page.
-  config.before(:each) { create(:casa_org) }
```

Comment:
I can add this back if we want, but the specs all still passed when I removed this, so let's start each test with a blank slate :)